run/logging-manual/main.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Sample Manual Logging shows how to leverage Cloud Run structured logging without a client library.
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"cloud.google.com/go/compute/metadata"
+)
+
+var projectID string
+
+func main() {
+	projectID = os.Getenv("GOOGLE_CLOUD_PROJECT")
+	if projectID == "" {
+		projectID, _ = metadata.ProjectID()
+	}
+	if projectID == "" {
+		log.Println("Could not determine Google Cloud Project. Running without log correlation. For local use set the $GOOGLE_CLOUD_PROJECT environment variable.")
```

Comment:
Is the dollar sign standard? I haven't seen it in other errors.

Reply:
Removing; I commonly include it from a BASH or PHP context.